GeneratorInterface/Hydjet2Interface/test/Hydjet2Analyzer.cc

```diff
@@ -160,7 +160,9 @@ class Hydjet2Analyzer : public edm::EDAnalyzer {
   edm::InputTag genParticleSrc_;
   edm::InputTag genHIsrc_;
   edm::InputTag simVerticesTag_;
-  edm::ESHandle<ParticleDataTable> pdt;
+  //edm::ESHandle<ParticleDataTable> pdt;
```

Comment:
I think we should remove the ESHandle line forever. GIT will keep history anyways, no need to keep it in the code.